build-tests/api-extractor-scenarios/etc/includeForgottenExports/api-extractor-scenarios.api.json

```diff
@@ -0,0 +1,638 @@
+          "members": [
+            {
+              "kind": "Class",
+              "canonicalReference": "api-extractor-scenarios!~ForgottenExport4~ForgottenExport5:class",
+              "docComment": "",
+              "excerptTokens": [
+                {
+                  "kind": "Content",
+                  "text": "class ForgottenExport5 "
+                }
+              ],
+              "releaseTag": "None",
+              "name": "ForgottenExport5",
+              "preserveMemberOrder": false,
+              "members": [],
+              "implementsTokenRanges": []
+            }
+          ]
+        },
+        {
+          "kind": "Namespace",
+          "canonicalReference": "api-extractor-scenarios!internal2:namespace",
+          "docComment": "",
+          "excerptTokens": [],
+          "releaseTag": "None",
+          "name": "internal2",
+          "preserveMemberOrder": false,
+          "members": [
+            {
+              "kind": "Class",
+              "canonicalReference": "api-extractor-scenarios!internal2.ForgottenExport6:class",
+              "docComment": "",
+              "excerptTokens": [
+                {
+                  "kind": "Content",
+                  "text": "export declare class ForgottenExport6 "
+                }
+              ],
+              "releaseTag": "None",
+              "name": "ForgottenExport6",
+              "preserveMemberOrder": false,
+              "members": [],
+              "implementsTokenRanges": []
+            }
+          ]
+        },
+        {
+          "kind": "Function",
+          "canonicalReference": "api-extractor-scenarios!someFunction1:function(1)",
+          "docComment": "/**\n * @public\n */\n",
+          "excerptTokens": [
+            {
+              "kind": "Content",
+              "text": "export declare function someFunction1(): "
+            },
+            {
+              "kind": "Reference",
+              "text": "ForgottenExport1",
+              "canonicalReference": "api-extractor-scenarios!~ForgottenExport1:class"
+            },
+            {
+              "kind": "Content",
+              "text": ";"
+            }
+          ],
+          "returnTypeTokenRange": {
+            "startIndex": 1,
+            "endIndex": 2
+          },
+          "releaseTag": "Public",
+          "overloadIndex": 1,
+          "parameters": [],
+          "name": "someFunction1"
+        },
+        {
+          "kind": "Function",
+          "canonicalReference": "api-extractor-scenarios!someFunction2:function(1)",
+          "docComment": "/**\n * @public\n */\n",
+          "excerptTokens": [
+            {
+              "kind": "Content",
+              "text": "export declare function someFunction2(): "
+            },
+            {
+              "kind": "Reference",
+              "text": "DuplicateName",
+              "canonicalReference": "api-extractor-scenarios!~DuplicateName:type"
```

Comment:
This canonical reference should be `api-extractor-scenarios!~DuplicateName_2:type`. This is the pre-existing bug 1 mentioned in the PR description.